Fix text for hypertext

pjessesco opened the pull request:
Change it same as another page. 
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-python
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-php
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-nodejs

azurecla commented:
Hi __@TheBeardedLlama__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@CawaMS - Another public repo update to review. Thanks

CawaMS commented:
#sign-off

pjessesco committed:
Fix text for hypertext

Change it same as another page.
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-python
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-php
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-get-started-nodejs
